src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,332 @@
+    entgeltp_ost: float,
+    entgeltp_zurechnungszeit: float,
+    anteil_entgeltp_ost: float,
+) -> float:
+    """Entgeltpunkte accumulated in Eastern Germany which Erwerbsminderungsrente
+    is based on (public disability insurance)
+    In the case of the public disability insurance,
+    pensioners are credited with additional earning points.
+    They receive their average earned income points for
+    each year between their age of retirement and the "zurechnungszeitgrenze".
+
+    Parameters
+    ----------
+    entgeltp_ost
+        See basic input variable :ref:`entgeltp_ost <entgeltp_ost>
+    entgeltp_zurechnungszeit
+        See :func:`entgeltp_zurechnungszeit`.
+    anteil_entgeltp_ost
+        See :func:`anteil_entgeltp_ost`.
+
+    Returns
+    -------
+    Final pension points for Erwerbsminderungsrente (public disability insurance)
+
+    """
+
+    out = entgeltp_ost + (entgeltp_zurechnungszeit * anteil_entgeltp_ost)
+
+    return out
+
+
+@dates_active(start="2001-01-01")
+def entgeltp_zurechnungszeit(
+    durchschn_entgeltp: float,
+    age_of_retirement: float,
+    erwerbsm_rente_params: dict,
+) -> float:
+    """Additional Entgeltpunkte accumulated through "Zurechnungszeit" for
+    Erwerbsminderungsrente (public disability insurance)
+    In the case of the public disability insurance,
+    pensioners are credited with additional earning points.
+    They receive their average earned income points for
+    each year between their age of retirement and the "zurechnungszeitgrenze".
+
+    Parameters
+    ----------
+    durchschn_entgeltp
+        See :func:`durchschn_entgeltp`.
+    age_of_retirement
+        See :func:`age_of_retirement`.
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+
+
+    Returns
+    -------
+    Final pension points for Erwerbsminderungsrente (public disability insurance)
+
+    """
+    zurechnungszeitgrenze = erwerbsm_rente_params["zurechnungszeitgrenze"]
+
+    out = (zurechnungszeitgrenze - (age_of_retirement)) * durchschn_entgeltp
+
+    return out
+
+
+@dates_active(start="2001-01-01")
+def rentenartfaktor(
+    teilw_erwerbsgemind: bool,
+    erwerbsm_rente_params: dict,
+) -> float:
+    """Rentenartfaktor for Erwerbsminderungsrente
+    (public disability insurance)
+
+    Legal reference: SGB VI § 67: Rentenartfaktor
+
+    Parameters
+    ----------
+    teilw_erwerbsgemind
+        See basic input variable :ref:`teilw_erwerbsgemind <teilw_erwerbsgemind>.
+    erwerbsm_rente_params
+        See params documentation :ref:`erwerbsm_rente_params <erwerbsm_rente_params>.
+
+    Returns
+    -------
+    Rentenartfaktor
+
+    """
+
+    if teilw_erwerbsgemind:
+        out = erwerbsm_rente_params["rentenartfaktor"]["teilw"]
+
+    else:
+        out = erwerbsm_rente_params["rentenartfaktor"]["voll"]
+
+    return out
+
+
+@dates_active(start="2001-01-01")
+def erwerbsm_rente_zugangsfaktor(
+    ges_rente_params: dict,
```

Comment:
Please always put all `params` last. Please don't forget to adjust the docstrings along with these changes.